src/main/java/dao/impl/WareHouseDaoImpl.java

```diff
@@ -0,0 +1,55 @@
+package dao.impl;
+
+import dao.WareHouseDao;
+import db.WareHouse;
+import java.util.Arrays;
+
+public class WareHouseDaoImpl implements WareHouseDao {
+    private static final int ZERO = 0;
```

Comment:
there's not much use in creating constants with names of the values
name variables after the role they serve, not after the value they have at the moment
This line gives no new valuable information to the reader, compare it to FRUIT_NAME_INDEX - much more useful